contrib/debian/copyright

```diff
@@ -76,8 +76,8 @@ Comment:
 
 Files: src/qt/res/icons/clock*.png
        src/qt/res/icons/eye_*.png
-       src/qt/res/icons/verify.png
        src/qt/res/icons/tx_in*.png
+       src/qt/res/icons/verify.png
```

Comment:
Nit: why does git think something changed here?

Reply:
I moved one element so it is alphabetical

Reply:
nit, unrelated change. If you happen to push again consider removing this change.